capa/features/extractors/viv/insn.py

```diff
@@ -120,6 +122,35 @@ def extract_insn_api_features(fh: FunctionHandle, bb, ih: InsnHandle) -> Iterato
                 yield API(name[1:]), ih.address
             return
 
+        if f.vw.metadata["Format"] == "elf":
+            if "SymbolTable" not in f.vw.metadata:
+                # the symbol table gets stored as a function's attribute in order to avoid running
+                # this code everytime the call is made, thus preventing the computational overhead.
+                elf = f.vw.parsedbin
+                endian = "<" if elf.getEndian() == 0 else ">"
+                bitness = elf.bits
+
+                SHT_SYMTAB = 0x2
+                for section in elf.sections:
+                    if section.vsGetField("sh_info") & SHT_SYMTAB != 0:
+                        strtab = elf.sections[section.vsGetField("sh_link")]
+                        sh_symtab = Shdr.from_viv(section, elf.getSectionBytes(section.name))
+                        sh_strtab = Shdr.from_viv(strtab, elf.getSectionBytes(strtab.name))
```

Comment:
how does viv handle the case when there are two sections with the same name?? i assume this is possible (but perhaps im wrong). in any case, let's avoid that ambiguity and read the section data as viv does internally:

https://github.com/vivisect/vivisect/blob/f81abdc8f9e0b37a5e6c9a84a77a4d75ff92d557/Elf/__init__.py#L691-L695

```suggestion
                        sh_symtab = Shdr.from_viv(section, elf.readAtOffset(section.sh_offset, section.sh_size))
                        sh_strtab = Shdr.from_viv(strtab, elf.readAtOffset(strtab.sh_offset, strtab.sh_size))
```